docs/resources/managed_user.md

```diff
@@ -1,44 +1,55 @@
 ---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "artifactory_managed_user Resource - terraform-provider-artifactory"
 subcategory: "User"
 ---
 # Artifactory Managed User Resource
 
 Provides an Artifactory managed user resource. This can be used to create and maintain Artifactory users. For example, service account where password is known and managed externally.
 
 Unlike `artifactory_unmanaged_user` and `artifactory_user`, the `password` attribute is required and cannot be empty.
-Consider using a separate provider to generate and manage passwords. 
+Consider using a separate provider to generate and manage passwords.
 
 ~> The password is stored in the Terraform state file. Make sure you secure it, please refer to the official [Terraform documentation](https://developer.hashicorp.com/terraform/language/state/sensitive-data).
 
 ## Example Usage
 
-```hcl
-# Create a new Artifactory user called terraform
+```terraform
```

Comment:
I've seen GH handle both of these declarations. I don't have a dog in this fight, but I'm curious why the change. Was `hcl` not working?